packages/eslint-config-universe/package.json

```diff
@@ -44,13 +43,15 @@
     "@typescript-eslint/parser": "^8.29.1",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-n": "^17.17.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^5.2.6",
     "eslint-plugin-react": "^7.37.5",
-    "eslint-plugin-react-hooks": "^5.2.0"
+    "eslint-plugin-react-hooks": "^5.2.0",
+    "globals": "^16.0.0"
   },
   "peerDependencies": {
-    "eslint": ">=8.10 <9",
+    "eslint": ">=9",
```

Comment:
Do we need to make peer dependencies check stricter? I would think that this will need to change, when we drop legacy config from the package.

Reply:
Oh yeah I guess it's `>=8.10`